python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -528,3 +534,51 @@ def _add_prefixes(names, prefixes):
         for i, (prefix, col_name) in enumerate(zip(prefixes, names)):
             prefixed_names[i] = f"{prefix}_{col_name}"
     return prefixed_names
+
+
+def _align_by_and_df(obj, by, how="inner"):
+    """
+    Returns a pair of dataframes and a list may be containing
```

Comment:
Perhaps we should include a couple of examples that make the purpose of this helper function clear?